app/api/chat/route.ts

```
@@ -117,22 +94,28 @@ async function generateWithFallback(
+    console.warn("Primary API key failed, trying secondary key:", error.message);
     
-    // Create a fallback response
-    return {
-      text: () => {
-        const language = contents.find(c => c.parts?.[0]?.text?.includes("IMPORTANT: You MUST respond exclusively in Kurdish"))
-          ? "ku"
-          : "en";
+    // Try with a simpler request format as fallback
+    try {
+      const lastUserMessage = contents.filter(c => c.role === 'user').pop();
+      if (lastUserMessage && lastUserMessage.parts && lastUserMessage.parts[0] && lastUserMessage.parts[0].text) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)